### PR TITLE
Add MuonGEMHits module as GEMSimHitsAnalyzer

### PR DESCRIPTION
**Pull request by geonmo**
I added MuonGEMHits module for simhit validation.

It was based on GEMValidation / scripts / produceSimHitValidationPlots.py and GEMCode / GEMValidation / plugins / GEMSimHitAnalyzer.cc.

I tested cmsDriver command, too.

### CODE REVIEW

File: Validation/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+*.root
+geonmo*
```
Comment (dildick) [inline Validation/.gitignore]: Why is this file included?

Comment (geonmo) [inline Validation/.gitignore]: It was just private code to use "commit -a".
I will remove that.

**Comment by jshlee** [comment]
checked
